duckduckgo/macos-browser
# Monitoring of data decryption added

Task/Issue URL: https://app.asana.com/0/1177771139624306/1203983976348012/f

**Description**:
This PR adds monitoring of the data decryption.

**Steps to test this PR**:
1. Make sure new pixels are in correct places and the app compiles and runs successfully. (Or you can try to simulate a decryption error)



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
